Src/FluentAssertions/Equivalency/AssertionRuleEquivalencyStep.cs

```diff
@@ -22,7 +22,7 @@ public bool CanHandle(IEquivalencyValidationContext context, IEquivalencyAsserti
         {
             Func<IMemberInfo, bool> predicate = canHandle.Compile();
 
-            return ((context.SelectedMemberInfo != null) && predicate(context));
+            return predicate(context);
```

Comment:
Can you elaborate a bit on the changes in this file?
I'm interested since no tests seem affected by this change.

Reply:
This step wasn't even used.

Reply:
Ahh... didn't realize that not used *at all* before.
Just checked out the branch locally to see that the changes in this file are necessary to pass the tests.